src/api/firebase.js

```diff
@@ -93,13 +96,31 @@ export async function updateItem(listId, itemData) {
 	 * this function must accept!
 	 */
 	const itemRef = doc(db, listId, itemData.id);
+	console.log('in updateItem');
+	let dayDiff;
+	if (itemData.dateLastPurchased) {
+		dayDiff = getDaysBetweenDates(itemData.dateLastPurchased);
+	} else {
+		dayDiff = getDaysBetweenDates(itemData.dateCreated);
+	}
```

Comment:
I wanna share an example of a really common syntax I see to conditionally assign a variable value in a ternary statement instead of an if/else code block so you have multiple tools in your toolkit

I show it with the original to show it gives the same console output && to test it to make sure my code works before I send it
![conditionalVariableAssignment](https://user-images.githubusercontent.com/89806097/183270261-aa392fca-4b95-4e95-a7d7-35db532ac2c6.PNG)

Reply:
Good work with the safety net of using dateCreated if there is no dateLastPurchased